src/main/java/mate/model/Manufacturer.java

```diff
@@ -49,4 +57,9 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id, name, country);
     }
+
+    @Override
+    public String toString() {
+        return "#" + id + " (" + name + ", " + country + ')';
```

Comment:
add identifier in your toString methods - some prefixes like `Manufacturer#`, to keep in logs understanding what is logged.